tensilelite/Tensile/TensileCreateLibrary.py

```diff
@@ -244,93 +230,83 @@ def buildSourceCodeObjectFile(CxxCompiler, outputPath, kernelFile):
 
     coFilenames = []
 
-    if supportedCompiler(CxxCompiler):
-      archs, cmdlineArchs = splitArchs()
+    archs, cmdlineArchs = splitArchs()
 
-      archFlags = ['--offload-arch=' + arch for arch in cmdlineArchs]
+    archFlags = ['--offload-arch=' + arch for arch in cmdlineArchs]
 
-      # needs to be fixed when Maneesh's change is made available
-      hipFlags = ["-D__HIP_HCC_COMPAT_MODE__=1"]
-      hipFlags += (
-          ["--genco"] if CxxCompiler == "hipcc" else ["--cuda-device-only", "-x", "hip", "-O3"]
-      )
-      # if CxxCompiler == "amdclang++":
-      # hipFlags += ["-mllvm", "-amdgpu-early-inline-all=true", "-mllvm", "-amdgpu-function-calls=false"]
+    # needs to be fixed when Maneesh's change is made available
+    hipFlags = ["-D__HIP_HCC_COMPAT_MODE__=1"]
+    hipFlags += ["--cuda-device-only", "-x", "hip", "-O3"]
+    hipFlags += ['-I', outputPath]
+    hipFlags += ["-Xoffload-linker", "--build-id=%s"%globalParameters["BuildIdKind"]]
+    hipFlags += ['-std=c++17']
+    if globalParameters["AsanBuild"]:
+      hipFlags += ["-fsanitize=address", "-shared-libasan", "-fuse-ld=lld"]
+    if globalParameters["SaveTemps"]:
+      hipFlags += ['--save-temps']
 
-      hipFlags += ['-I', outputPath]
-      hipFlags += ["-Xoffload-linker", "--build-id=%s"%globalParameters["BuildIdKind"]]
-      hipFlags += ['-std=c++17']
-      if globalParameters["AsanBuild"]:
-        hipFlags += ["-fsanitize=address", "-shared-libasan", "-fuse-ld=lld"]
-      if globalParameters["SaveTemps"]:
-        hipFlags += ['--save-temps']
+    launcher = shlex.split(os.environ.get('Tensile_CXX_COMPILER_LAUNCHER', ''))
```

Comment:
Can we do this once when we validate the toolchain?

Reply:
Out of scope. This behaviour will be added in the subsequent PR.